### ISSUE TITLE
occ doesn't support enabling/disabling multiple apps at once

### ISSUE THREAD

**Issue by robert-scheck** [nextcloud/server#11274]
As of Nextcloud 14.0.0, `occ` still doesn't support enabling/disabling multiple apps at once:

```
$ ./occ --no-interaction app:enable richdocuments spreed

                                                              
  Too many arguments, expected arguments "command" "app-id".  
                                                              

app:enable [-g|--groups GROUPS] [--] <app-id>

$
```

```
$ ./occ --no-interaction app:disable richdocuments spreed

                                                              
  Too many arguments, expected arguments "command" "app-id".  
                                                              

app:disable <app-id>

$ 
```

Given that usage of `occ` is just slow (~ 0.75-1.25 seconds here), enabling/disabling apps takes a lot of time, especially in containers where e.g. some apps always should be (re-)enabled.

```
$ time ./occ --no-interaction app:enable richdocuments
richdocuments enabled

real	0m1.167s
user	0m0.632s
sys	0m0.407s
$ 
```

```
$ time ./occ --no-interaction app:disable richdocuments
richdocuments disabled

real	0m0.746s
user	0m0.546s
sys	0m0.182s
$ 
```

Yes, you could perform the steps on the database directly, but that likely isn't supported nor future-safe.

**Comment by robert-scheck** [comment]
This suggestion could be maybe less relevant if #11275 is being implemented instead.

**Comment by learnsomething2** [comment]
With the latest upgrade to 24.0.4 I got an error something went not good while upgrading, out of my experience it's belonging to apps they are not supporting the newer versions of Nextcloud. I was looking for a way to deactivate them all in one command.
0. `sudo -u www-data php /var/www/nextcloud/occ maintenance:mode --off`
1.  I did a list `sudo -u www-data php /var/www/nextcloud/occ app:list >> ~/app.list` 
2. I edit the list and took out all the version numbers and listing singes
3. I wehre running the list througt a variable with `for i in $(cat ~/app.list) do  sudo -u www-data php /var/www/nextcloud/occ app:disable $i; done`
4. and than `sudo -u www-data php /var/www/nextcloud/occ upgrade` agian

**Comment by carloscomputer** [comment]
With the latest upgrade to 24.0.4 I got an error something went not good while upgrading, out of my experience it's belonging to apps they are not supporting the newer versions of Nextcloud. I was looking for a way to deactivate them all in one command.

0. `sudo -u www-data php /var/www/nextcloud/occ maintenance:mode --off`
1.  I did a list `sudo -u www-data php /var/www/nextcloud/occ app:list >> ~/app.list` 
2. I edit the list and took out all the version numbers and listing singes
3. I wehre running the list througt a variable with `for i in $(cat ~/app.list) do  sudo -u www-data php /var/www/nextcloud/occ app:disable $i; done`
4. and than `sudo -u www-data php /var/www/nextcloud/occ upgrade` agian
